ui-participant/src/hub/documents/DocumentLibrary.tsx

```diff
@@ -20,10 +27,20 @@ export default function DocumentLibrary() {
     pStudy.study.studyEnvironments.find(studyEnv =>
       studyEnv.environmentName === portalEnv.environmentName))?.study
 
+  if (!currentStudy) {
+    return <div>Study not found</div>
+  }
+
+  const studyEnvParams: StudyEnvParams = {
+    portalShortcode: portal.shortcode,
+    studyShortcode: currentStudy?.shortcode,
+    envName: portalEnv.environmentName as EnvironmentName
+  }
+
   const loadDocuments = async () => {
```

Comment:
For some reason, this isn't loading properly. It doesn't seem to ever actually make any network calls to load the files.

<img width="723" alt="image" src="https://github.com/user-attachments/assets/513622fd-9108-46bd-a271-933984be23a6" />

Reply:
I'm having trouble reproducing this issue 🤔